incubator/keycloak/values.yaml

```diff
@@ -43,6 +43,11 @@ keycloak:
   nodeSelector: {}
   tolerations: []
 
+  livenessInitialDelaySeconds: 120
+  livenessTimeoutSeconds: 5
+  readinessInitialDelaySeconds: 30
+  readinessTimeoutSeconds: 1
+
```

Comment:
would about putting these under a `probes` object?

Reply:
I'd suggest you do this:

```yaml
livenessProbe:
  initialDelaySeconds: 120
  timeoutSeconds: 5
readinessProbe:
  initialDelaySeconds: 30
  timeoutSeconds: 1
```